pennylane_lightning/core/src/bindings/Bindings.hpp

```diff
@@ -350,6 +350,8 @@ void registerBackendAgnosticObservables(py::module_ &m) {
         .def("__repr__", &HermitianObs<StateVectorT>::getObsName)
         .def("get_wires", &HermitianObs<StateVectorT>::getWires,
              "Get wires of observables")
+        .def("get_matrix", &HermitianObs<StateVectorT>::getMatrix,
```

Comment:
I don't mind adding this, but where are these new bindings used?

Reply:
Lee added these so that we can debug serialization issues more easily. We were running into a serialization issue where the actual and expected serialization had the exact same repr, but the test asserting equivalence between the two was failing, and because this goes into the C++ layer it was really hard to debug.

Reply:
This was required to debug changes in the tests for the serialisation process, and is a low-hanging fruit that should've been added previously. No harm keeping this in.